### PR TITLE
Repurpose 'Info' to mean 'Suggestion'.

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**

### CODE REVIEW

**Comment by CyrusNajmabadi** [comment]
As per our design meeting today, we're going to use hte existing 'Info' severity to correspond to Suggestions.

Tagging @dotnet/roslyn-ide

**Comment by CyrusNajmabadi** [comment]
test vsi please

**Comment by gafter** [comment]
I like this, particularly when compared to the previously considered approach.

**Comment by CyrusNajmabadi** [comment]
retest windows_debug_unit32_prtest  please

**Comment by CyrusNajmabadi** [comment]
retest windows_vsi_p0_open_prtest  please

**Comment by CyrusNajmabadi** [comment]
retest windows_vsi_p2_open_prtest  please

**Comment by CyrusNajmabadi** [comment]
retest windows_vsi_p3_open_prtest  please

File: src/EditorFeatures/Core/Implementation/Adornments/GraphicsTag.cs [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+                return;
+            }
+
+            // TODO: Refresh this when the user changes fonts and colors
+
+            // TODO: Get from resources
+            var lightGray = Color.FromRgb(0xE0, 0xE0, 0xE0);
+
+            var outliningForegroundBrush = view.VisualElement.TryFindResource("outlining.verticalrule.foreground") as SolidColorBrush;
+            var color = outliningForegroundBrush?.Color ?? lightGray;
```
Comment (Pilchie) [inline src/EditorFeatures/Core/Implementation/Adornments/GraphicsTag.cs]: We should test this in high-contrast themes, in addition to the dark theme.

File: src/EditorFeatures/Core/Implementation/Diagnostics/SuggestionTag.cs [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        }
+
+        public override GraphicsResult GetGraphics(IWpfTextView view, Geometry geometry)
+        {
+            Initialize(view);
+
+            // We clip off a bit off the start of the line to prevent a half-square being
+            // drawn.
+            var clipRectangle = geometry.Bounds;
+            clipRectangle.Offset(1, 0);
```
Comment (Pilchie) [inline src/EditorFeatures/Core/Implementation/Diagnostics/SuggestionTag.cs]: This doesn't seem like it should be necessary - can we get a WPF expert to weigh in here?

**Comment by Pilchie** [comment]
I still think it would be great to add support for this directly into the editor's adornment layer via `IErrorTag`, but I'm happy to check this in so we can start using it and see how it feels in the meantime.

We should follow up with a change to bump some of our `Hidden` diagnostics into `Info` ones, but otherwise :shipit:

**Comment by CyrusNajmabadi** [comment]
test vsi please

**Comment by CyrusNajmabadi** [comment]
test vsi please

**Comment by CyrusNajmabadi** [comment]
retest vsi please

**Comment by CyrusNajmabadi** [comment]
retest vsi please

**Comment by CyrusNajmabadi** [comment]
retest vsi please

**Comment by CyrusNajmabadi** [comment]
retest windows_vsi_p1_open_prtest  please

**Comment by CyrusNajmabadi** [comment]
retrest windows_vsi_p1_open_prtest  please

**Comment by CyrusNajmabadi** [comment]
retest windows_vsi_p1_open_prtest  please

**Comment by CyrusNajmabadi** [comment]
retest perf_correctness_prtest  please

**Pull request merged**